create people page

SK-143381 opened the pull request.

SK-143381 committed:
create people page

jooyoungseo commented:
@SK-143381 Thanks for drafting this. What's the notebook file there?

jooyoungseo commented on people.qmd:
```diff
@@ -0,0 +1,131 @@
+
+<div class="card mb-3">
+<div class="card-body">
+<img src="assets/images/js.png"/>
+<h3 class="card-title">JooYoung Seo</h3>
```
Could you please upload our headshot assets there, including yours and Aziz's? Mine can be found at my faculty site.

SK-143381 committed:
docs: :memo: add headshots and update page

added individual images and shifted content of the pages.

SK-143381 committed:
style: :lipstick: fix index page, projects page and people page

The home page now contains the lab logo in the background. The lab welcome message has been centered to the bottom of the screen. The extra sections from the people page have been commented currently. The Projects page has been updated with official funded projects.

jooyoungseo reviewed (approved):
LGTM!
